src/rules/no-named-default.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  meta: {
+    docs: {},
+  },
+
+  create: function (context) {
+    return {
+      'ImportDeclaration': function (node) {
+        if (node.source == null) return
```

Comment:
TBH, this is here because other rules included it (`named`, `no-deprecated`). It now seems likely that it's not needed here, will remove.